modules/cli/src/main/java/org/apache/ignite/cli/core/call/CallExecutionPipeline.java

```diff
@@ -90,24 +90,25 @@ public static <I extends CallInput, T> CallExecutionPipelineBuilder<I, T> builde
         return new CallExecutionPipelineBuilder<>(call);
     }
 
-    /** {@inheritDoc} */
-    public void runPipeline() {
+    /**
+     * Runs the pipeline.
+     *
+     * @return exit code.
+     */
+    public int runPipeline() {
         I callInput = inputProvider.get();
 
         CallOutput<T> callOutput = call.execute(callInput);
 
         if (callOutput.hasError()) {
-            exceptionHandlers.handleException(ExceptionWriter.fromPrintWriter(errOutput), callOutput.errorCause());
-            return;
+            return exceptionHandlers.handleException(ExceptionWriter.fromPrintWriter(errOutput), callOutput.errorCause());
         }
 
-        if (callOutput.isEmpty()) {
-            return;
+        if (!callOutput.isEmpty()) {
```

Comment:
Why did you decided to revert the expresion? Now it is less readable but I can't get the reason.

Reply:
Because otherwize two return statements will be needed and this would be less readable than negative condition IMO.